Extension/package.json

```diff
@@ -470,6 +470,15 @@
           "description": "The value to use for the system include path. If set, it overrides the system include path acquired via \"compilerPath\" and \"compileCommands\" settings.",
           "scope": "resource"
         },
+        "C_Cpp.default.enableConfigurationSquiggles": {
+          "type": [
+            "boolean",
+            "null"
+          ],
+          "default": null,
+          "description": "Set to false to disable squiggles for c_cpp_properties.json.",
```

Comment:
"Controls whether the extension will report errors detected in c_cpp_properties.json"